### ISSUE TITLE
Can't seem to select the theme in Atom 1.7.3

### ISSUE THREAD

**Issue by CoWinkKeyDinkInc** [atom-material/atom-material-ui#284]
I go to the drop down menu UI and Syntax Theme, and both material-ui and material-syntax don't show up, even though they're both installed.

What do I do?

**Comment by silvestreh** [comment]
I'm not sure it's a problem with the theme. Are you sure you're not running Atom in safe mode?

**Comment by CoWinkKeyDinkInc** [comment]
It isn't in safe mode, other packages I have installed are still running.

**Comment by silvestreh** [comment]
That's odd… is the dev console printing any errors or warnings? Check the CONTRIBUTING file, it'll help you make a better bug report :)

**Comment by silvestreh** [comment]
Did you work this out?

**Comment by silvestreh** [comment]
I'm closing this due to inactivity

**Comment by christoga** [comment]
If you want to open an issue, better be active.
_You probably should put that in the README_

**Comment by CoWinkKeyDinkInc** [comment]
I'm very sorry about my inactivity, but the issue seemed to resolve itself somehow.  I'm still on the same version.

**Comment by silvestreh** [comment]
@christoga, you're not helping with that attitude. Please understand that I'm not getting paid to work on this theme.

Here's how it works: I asked about this issue 5 days ago and, in the absence of a reply, I have to assume he either resolved it or wasn't experiencing the problem anymore, hence, I closed the issue.

I close inactive issues so the issue tracker doesn't become a mess and I can prioritize and work on them one by one.

In other words: a clean issue tracker helps me focus on the real problems.

**Comment by silvestreh** [comment]
Also @CoWinkKeyDinkInc no need to be sorry, you didn't do anything wrong. As I tried to explain above, it's not you, it's just me trying to organize the repo 😁 . I always welcome any kind of contribution, be it code, bug reports, or feature requests.
